src/File.php

```diff
@@ -243,16 +243,16 @@ private function readStream(bool $send): void
 
             $this->uncompressedSize += strlen($data);
 
-			if($deflate) {
-				/** @psalm-suppress InvalidArgument */
-				$data =  deflate_add(
-					$deflate,
-					$data,
-					$this->stream->eof()
-						? ZLIB_FINISH
-						: ZLIB_NO_FLUSH
-				);
-			}
+            if ($deflate) {
+                /** @psalm-suppress InvalidArgument */
+                $data =  deflate_add(
+                    $deflate,
+                    $data,
+                    $this->stream->eof()
+                        ? ZLIB_FINISH
+                        : ZLIB_NO_FLUSH
+                );
+            }
```

Comment:
Huh, usually PHPStorm picks the indentation based on the file for me automatically. I don't recall ever having a problem before 😆

Reply:
Hm. maybe a `.editorconfig` would help?